src/messaging/messaging.service.ts

```
@@ -0,0 +1,51 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Message } from './entities/message.entity';
+
+@Injectable()
+export class MessagingService {
+  constructor(
+    @InjectRepository(Message)
+    private messageRepository: Repository<Message>,
+  ) {}
+
+  async create(senderId: string, receiverId: string, message: string): Promise<Message> {
```

ESLint

[error] Replace `senderId:·string,·receiverId:·string,·message:·string` with `⏎····senderId:·string,⏎····receiverId:·string,⏎····message:·string,⏎··`

(prettier/prettier)